packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjects.js

```diff
@@ -0,0 +1,68 @@
+import { Anchor, Box, ResponsiveContext, Text } from 'grommet'
+import { arrayOf, bool, shape, string } from 'prop-types'
+import { useContext } from 'react'
+import { Loader, SpacedHeading, SpacedText } from '@zooniverse/react-components'
+
+import SubjectCard from '../SubjectCard/SubjectCard.js'
+
+function RecentSubjects({ isLoading = false, subjects = [] }) {
+  const size = useContext(ResponsiveContext)
+
+  return (
+    <Box
+      pad='medium'
+      height={{ min: '200px' }}
+      round='small'
+      border={{
+        color: { light: 'light-5', dark: 'black' },
+        size: 'xsmall'
+      }}
+    >
+      <SpacedHeading size='1.125rem' level={2} margin={{ top: '0' }}>
+        Recent Classifications
+      </SpacedHeading>
+      <Box
+        as='ul'
+        direction='row'
+        gap='small'
+        pad={{ horizontal: 'xxsmall', bottom: 'xsmall' }}
+        overflow={{ horizontal: 'auto' }}
+        style={{ listStyle: 'none' }}
+        margin='0'
+      >
+        {isLoading && (
+          <Box fill justify='center' align='center'>
+            <Loader />
+          </Box>
+        )}
+        {!isLoading && !subjects?.length && (
+          <Box fill justify='center' align='center' pad='medium'>
+            <SpacedText>No Recent Classifications found</SpacedText>
+            <Text>
+              Start by{' '}
+              <Anchor href='https://www.zooniverse.org/projects'>
+                classifying any project
+              </Anchor>{' '}
+              to show your recent classifications here.
+            </Text>
+          </Box>
+        )}
+        {!isLoading && subjects?.length
+          ? subjects
+              .slice(0, 10)
+              .map(subject => <SubjectCard key={subject.id} size={size} subject={subject} />)
+          : null}
```

Comment:
[super minor][coding] This uses the ternary so we have `!isLoading && subjects?.length ? subjects.slice().map(...) : null`, but wouldn't the rendered effects still be the same if we write it as `!isLoading && subjects?.slice().map(...)` ? I thought running a render-mapping an empty array was valid - i.e. `[].map(x => <div>{x}</div)` returns `[]`, which React renders as nothing but doesn't break.

Actually, you know what? Shaun TODO: double check this. A quick search for "React render empty array" on the Goog shows differing opinions on the interwebs.